src/main/java/org/springframework/data/elasticsearch/core/ElasticsearchTemplate.java

```diff
@@ -931,7 +930,13 @@ private SearchResponse doSearch(SearchRequestBuilder searchRequest, SearchQuery
 				searchRequest.addAggregation(aggregatedFacet.getFacet());
 			}
 		}
-		return getSearchResponse(searchRequest.setQuery(searchQuery.getQuery()).execute());
+		return getSearchResponse(searchRequest.setQuery(searchQuery.getQuery()));
+	}
+
+	private SearchResponse getSearchResponse(SearchRequestBuilder requestBuilder) {
+		if (QUERY_LOGGER.isDebugEnabled())
+			QUERY_LOGGER.debug(requestBuilder.toString());
```

Comment:
I'm not sure if Spring has any guidelines for this, but I found this old document recommending to include curly-braces around if statements - http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-142311.html